core/cap-0046-07.md

```diff
@@ -289,11 +289,22 @@ LedgerDataAccess_fee(resources) =
 where `BucketListSize` is the average size of the bucket list over the moving window. Refer to the [State Archival CAP](cap-0046-12.md) for details, and `write_fee_per_1kb` is a function that determines the ledger write fee per 1024 bytes based on the bucket list size and is defined as follows:
 
 ```
-write_fee_per_1kb(s) = (writeFee1KBBucketListHigh - writeFee1KBBucketListLow)*min(s, bucketListTargetSizeBytes)/bucketListTargetSizeBytes +
-  writeFee1KBBucketListLow +
-  (if s > bucketListTargetSizeBytes,
-      bucketListWriteFeeGrowthFactor*
-      (writeFee1KBBucketListHigh - writeFee1KBBucketListLow)*
+// this is the fee rate slope
+// feeRate1KB = (writeFee1KBBucketListHigh - writeFee1KBBucketListLow)/bucketListTargetSizeBytes
+
+// in all cases, rate is clamped as to not fall under MINIMUM_WRITE_FEE_PER_1KB in case
+// writeFee1KBBucketListLow or writeFee1KBBucketListHigh are too low
+
+// if s < bucketListTargetSizeBytes,
+//   grow by feeRate1KB until we reach writeFee1KBBucketListHigh
```

Comment:
Should this say "reach bucketListTargetSizeBytes" instead of "reach writeFee1KBBucketListHigh"? Or did you mean grow until write_fee_per_1kb reaches writeFee1KBBucketListHigh - writeFee1KBBucketListLow?

Reply:
Yes the "y" axis is the fee so we reach "high" when "s" is target size